tools/fv_diagnostics.F90

```diff
@@ -1140,7 +1150,10 @@ subroutine fv_diag_init(Atm, axes, Time, npx, npy, npz, p_ref)
             'Vertically Integrated Snow', 'kg/m**2', missing_value=missing_value )
        id_intqg = register_diag_field ( trim(field), 'intqg', axes(1:2), Time,        &
             'Vertically Integrated Graupel', 'kg/m**2', missing_value=missing_value )
-
+       id_inthght = register_diag_field ( trim(field), 'inthght', axes(1:2), Time,        &
```

Comment:
Are inthght and inttemp supposed to be proportional to the vertically-integrated potential and internal energies?

Reply:
Yes, that's correct. This could be overt by multiplying by the relevant constants 'grav' and 'cp_air' that are available in the module and changing the unit accordingly.